### PR TITLE
More unittests and multiple bugfixes

### PR DESCRIPTION
**Pull request by tsvi**
This adds unittests and bugfixes for:
- get_omer_string (bugfix see #7)
- hebrew_number
- get_holiday
- Remove case of Zhabotinsky day falling on Sabbath as this is impossible

### CODE REVIEW

File: hdate/__init__.py [hunk 1]
```diff
@@ -402,9 +396,6 @@ def get_reading(self, diaspora):
             return 0
         elif self._h_weeks == 4:
             if self._h_new_year_weekday == 7:
-                # Simhat tora in israel
```
Comment (royi1000) [inline hdate/__init__.py]: why did you deleted this?

File: hdate/__init__.py [hunk 2]
```diff
@@ -491,11 +482,8 @@ def get_reading(self, diaspora):
                 else:
                     reading += 1
             if (JOIN_FLAGS[diaspora][self._h_year_type - 1][6] and
-                    (reading >= 51)):
```
Comment (royi1000) [inline hdate/__init__.py]: we don't need reading += 1 here?

**Comment by tsvi** [comment]
There is no input which can cause the function to reach these cases.

File: hdate/__init__.py (hunk 1 above)
Comment (tsvi) [reply hdate/__init__.py]: Impossible to reach this case.
Unittests cover all possible years. This case is never reached.

File: hdate/__init__.py (hunk 2 above)
Comment (tsvi) [reply hdate/__init__.py]: Same as above.

Impossible to reach this case.
Unittests cover all possible years. This case is never reached.

**Pull request merged**